skaffold.yaml

```diff
@@ -54,8 +54,6 @@ deploy:
               k8sMasterPort: 0
         overrides:
           distributor:
-            image:
```

Comment:
Any reason why we are removing the version here?

Reply:
it's already defined in the chart/values.yaml, this information is therefore redundant.